cmake/templates/_setup_util.py.in

```diff
@@ -265,8 +265,11 @@ if __name__ == '__main__':
         CMAKE_PREFIX_PATH = '@CMAKE_PREFIX_PATH_AS_IS@'.split(';')
         # prepend current workspace if not already part of CPP
         base_path = os.path.dirname(__file__)
-        if base_path not in CMAKE_PREFIX_PATH:
-            CMAKE_PREFIX_PATH.insert(0, base_path)
+
+        # CMAKE_PREFIX_PATH uses forward slash on all platforms, but os.path.dirname is platform dependent
```

Comment:
> but os.path.dirname is platform dependent

This should be change to: `but __file__ is platform dependent`

Reply:
will change to `but __file__ is platform dependent`